certdb/mysql/migrations/001_CreateCertificates.sql

```diff
@@ -7,8 +7,8 @@ CREATE TABLE certificates (
   ca_label                 varbinary(128),
   status                   varbinary(128) NOT NULL,
   reason                   int,
-  expiry                   timestamp DEFAULT '0000-00-00 00:00:00',
-  revoked_at               timestamp DEFAULT '0000-00-00 00:00:00',
+  expiry                   timestamp,
```

Comment:
Why are these changes required? They seem to be causing a different failure:
```
$ go install ./vendor/bitbucket.org/liamstask/goose/cmd/goose
0.28s$ if [[ $(uname -s) == 'Linux' ]]; then psql -c 'create database certdb_development;' -U postgres; goose -path $GOPATH/src/github.com/cloudflare/cfssl/certdb/pg up; mysql -e 'create database certdb_development;' -u root; goose -path $GOPATH/src/github.com/cloudflare/cfssl/certdb/mysql up; fi
CREATE DATABASE
goose: migrating db environment 'development', current version: 0, target: 1
OK    001_CreateCertificates.sql
goose: migrating db environment 'development', current version: 0, target: 1
2019/05/09 01:13:57 FAIL 001_CreateCertificates.sql (Error 1067: Invalid default value for 'revoked_at'), quitting migration.
The command "if [[ $(uname -s) == 'Linux' ]]; then psql -c 'create database certdb_development;' -U postgres; goose -path $GOPATH/src/github.com/cloudflare/cfssl/certdb/pg up; mysql -e 'create database certdb_development;' -u root; goose -path $GOPATH/src/github.com/cloudflare/cfssl/certdb/mysql up; fi" failed and exited with 1 during .
Your build has been stopped.
```

Reply:
Yes, from what I can tell the version of mysql that travis is currently using is conflicting with the previous definition of that table.
```
Error 1067: Invalid default value for 'revoked_at'), quitting migration.
```
This appears to be related to a change with mysql 5.7 and discussed here:
https://stackoverflow.com/questions/9192027/invalid-default-value-for-create-date-timestamp-field